hack/install.sh

```diff
@@ -14,27 +14,16 @@
 # limitations under the License.
 
 
-# The git command will fetch the most recent tag across all branches
-LATEST_TAG=$(git describe --tags $(git rev-list --tags --max-count=1))
-NAMESPACE=nexus
+VERSION=$1
 
-echo "[INFO]  The repository will be checked out at the latest release"
-echo "....... Checkout code at ${LATEST_TAG} ......"
-git checkout tags/${LATEST_TAG}
+if [ -z ${VERSION} ]; then
+    echo "Please inform the desired version"
+    exit(1)
+fi
```

Comment:
The previous alternative was counting on having this repo cloned locally, that's why it worked. With the new approach you won't have to clone anything. Actually this script is basically useless now since a simple `kubectl apply -f <release/nexus-operator.yaml>` will do.

Reply:
+1 to removing the script altogether :-)